pkgs/by-name/ws/wsjtz/package.nix

```
@@ -0,0 +1,31 @@
+{
+  wsjtx,
+  fetchzip,
+  lib,
+}:
+
+wsjtx.overrideAttrs (old: rec {
+  pname = "wsjtz";
+  version = "2.7.0-rc7-1.43";
+
+  src = fetchzip {
+    url = "https://psychz.dl.sourceforge.net/project/wsjt-z/Source/wsjtz-${version}.zip";
+    hash = "sha256-m+P83S5P9v3NPtifc+XjZm/mAOs+NT9fTWXisxuWtZo=";
```

```suggestion
    url = "mirror://sourceforge/wsjt-z/Source/wsjtz-${version}.zip";
    hash = "sha256-m+P83S5P9v3NPtifc+XjZm/mAOs+NT9fTWXisxuWtZo=";
```
I think the mirror scheme is preferred for sourceforge.

---

Fixed (: